app/src/main/AndroidManifest.xml

```diff
@@ -7,18 +7,26 @@
     android:allowBackup="true"
     android:icon="@mipmap/ic_launcher"
     android:label="@string/app_name"
+    android:name="com.example.myapplication.OppiaApplication"
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
     android:theme="@style/OppiaTheme">
-    <activity android:name="org.oppia.app.HomeActivity">
+    <activity android:name=".HomeActivity">
```

Comment:
Isn't this already declared in this manifest?

Reply:
its been removed.